pennylane/templates/broadcast.py

```diff
@@ -25,7 +25,7 @@
 from pennylane.templates.utils import get_shape
 from pennylane.wires import Wires
 
-OPTIONS = ["single", "double", "double_odd", "chain", "ring", "pyramid", "all_to_all", "custom"]
+OPTIONS = {"single", "double", "double_odd", "chain", "ring", "pyramid", "all_to_all", "custom"}
```

Comment:
is this just for faster 'contains' checking?

Reply:
Yes. By making it a set makes the `if x in OPTIONS` time O(1)